spec/atlas/dataset/derived_dataset_spec.rb

```diff
@@ -32,47 +32,102 @@ module Atlas; describe Dataset::Derived do
   end
 
   describe '(validations)' do
-    before do
-      dataset.init = init
-      dataset.valid?
-    end
+    describe "init" do
+      before do
+        dataset.init = init
+        dataset.valid?
```

Comment:
If you use the `errors_on` matcher, you don't need to call `valid?` explicitly; [it does it for you](https://github.com/quintel/atlas/blob/0ccffb6b3f7a7240d5badceade37676e14f38781/spec/support/matchers.rb#L43).